src/extension.ts

```diff
@@ -176,6 +180,19 @@ function initializeIconPaths(context: vscode.ExtensionContext) {
     ext.iconPaths.light.schema = context.asAbsolutePath('resources/light/schema.svg')
 }
 
+function updateMruWhenAwsContextChanges(awsContext: AwsContext, extensionContext: vscode.ExtensionContext) {
```

Comment:
Is extension.ts the best place for this function? I agree that the defaultAwsContext isn't, but it doesn't seem explicitly tied to plugin load.

Reply:
Agreed - What do you think about a dedicated activation file for credentials? This aligns with upcoming credentials changes.